src/main/java/pro/sky/telegrambot/model/NotificationTask.java

```diff
@@ -0,0 +1,80 @@
+package pro.sky.telegrambot.model;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+@Entity
+public class NotificationTask {
+    @Id
+    @GeneratedValue
```

Comment:
```suggestion
    @GeneratedValue(strategy = GenerationType.IDENTITY)
```
Нужно указать стратегию.